client/components/LearnerViewDirective.js

```diff
@@ -37,99 +37,103 @@ tie.directive('learnerView', [function() {
                 </div>
               </div>
             </div>
-            <div class="tie-coding-ui">
-              <div class="tie-feedback-window" ng-class="{'night-mode': isInDarkMode}">
-                <div class="tie-feedback">
-                  <p ng-repeat="paragraph in feedbackParagraphs track by $index"
-                      class="tie-feedback-paragraph"
-                      ng-class="{'tie-feedback-paragraph-code': paragraph.isCodeParagraph()}">
-                    <span ng-if="$first">{{feedbackTimestamp}}</span>
-                    <span ng-if="paragraph.isTextParagraph()">
+            <div class="tie-question-ui">
+              <div class="tie-question-window" ng-class="{'night-mode': isInDarkMode}">
+                <div class="tie-greetings">
```

Comment:
Recommend keeping 'feedback' nomenclature, as the feedback window will more often contain feedback than greetings. Same for other instanced of 'greeting' below.